src/lib/structures/CommandMessage.js

```diff
@@ -172,7 +172,7 @@ class CommandMessage {
 	 */
 	static getArgs(cmdMsg) {
 		// eslint-disable-next-line newline-per-chained-call
-		const args = cmdMsg.msg.content.slice(cmdMsg.prefixLength).trim().split(' ').slice(1).join(' ').trim().split(cmdMsg.cmd.usageDelim !== '' ? cmdMsg.cmd.usageDelim : undefined);
+		const args = cmdMsg.msg.content.slice(cmdMsg.prefixLength).trim().split(/\s+/g).slice(1).join(' ').trim().split(cmdMsg.cmd.usageDelim !== '' ? cmdMsg.cmd.usageDelim : undefined);
```

Comment:
Does the first split remove the command name? I see the second split splits on usageDelim.

Reply:
This has a serious flag: it'll work perfectly when you use space as usage delim, but it will not when the usageDelim is different, causing weird issues. Same goes for quoted string arguments.